its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -589,7 +588,8 @@ private void runBeginBuildAndEndForStandardProject(String folderName, Boolean se
       .addArgument("begin")
       .setProjectKey(folderName)
       .setProjectName(folderName)
-      .setProjectVersion("1.0");
+      .setProjectVersion("1.0")
+      .setProperty("sonar.sourceEncoding", "UTF-8");
```

Comment:
This is the real fix, for some reason the file is now detected as `windows-1252` which doesn't match Roslyn detected encoding (`utf-8`).